src/Tasks.UnitTests/Microsoft.Build.Tasks.UnitTests.csproj

```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="System.Security.Principal.Windows" />
     <PackageReference Include="System.Xml.XPath" />
     <PackageReference Include="Shouldly" />
+    <PackageReference Include="NuGet.CommandLine" VersionOverride="4.4.3" GeneratePathProperty="true" />
```

Comment:
I didn't want to affect other NuGet versions if we import it intentionally at some point. I imagine we'd generally want a modern NuGet, and this is the exception.

Reply:
a few other options:
- Use PackageDownload Version=[4.4.3] -- instead of PackageReference
- download package directly from nuget.org/downloads url.
- other???

both of these would keep the package out of the restore graph...and likely avoid problems in the future.

Reply:
When would those be evaluated? The initial reason I went with this approach is that @rainersigwald wanted to avoid possible flakiness from a test needing to download something. If all the downloads happen at restore time, it contains the time in which the tests need a consistent internet connection.

Reply:
packagedownload item in msbuild would happen at restore time, just like PackageReference...but would have less impact on the restore graph created in obj\project.assets.json.

Reply:
PackageDownload is pretty ideal for this case. Good idea!